### PR TITLE
fix People tab icon for organizations

### LINKED ISSUE #13310: Update steps described in: Removing a member from your organization
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

[Removing a member from your organization](https://docs.github.com/en/organizations/managing-membership-in-your-organization/removing-a-member-from-your-organization)

### What part(s) of the article would you like to see updated?

[Revoking the user's membership](https://docs.github.com/en/organizations/managing-membership-in-your-organization/removing-a-member-from-your-organization#revoking-the-users-membership)

### Additional information

Step 3 describes steps from now-outdated UI ([outdated screenshot](https://docs.github.com/assets/cb-17298/images/help/organizations/organization-people-tab-with-overview-tab.png))

### PR DESCRIPTION
**Pull request by AnilSeervi**
### Why:

Closes #13310 

### What's being changed:
- People tab was described using the **organization** icon; Use **Octicon Person** as denoted in the UI.

Preview - https://docs-13313--fix-people-icon.herokuapp.com/en/organizations/managing-membership-in-your-organization/removing-a-member-from-your-organization
### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@AnilSeervi 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Review by skedwards88 — APPROVED** [review]
Thanks for this fix! Our repo is currently frozen until early January, but we will merge this for you once the freeze lifts.

**Comment by banagale** [comment]
@ramyaparimi Thank you, Ramya!

**Pull request merged**
